Integrations/integration-PagerDuty.yml

```diff
@@ -124,16 +122,18 @@ script:
             'ReadableContentsFormat': formats['markdown'],
             'HumanReadable': tableToMarkdown(USERS_ON_CALL, outputs, USERS_ON_CALL_NOW_HEADERS),
             'EntryContext': {
-                'PagerDutyUser': context
+                'PagerDutyUser(true)': contexts
```

Comment:
@ronykoz why is the condition equals to `true`? should be `PagerDutyUser(val.ID==obj.ID)`

Reply:
Wanted it to overwrite all the previous data